frontend/src/scenes/error-tracking/issue/OccurenceSparkline.tsx

```diff
@@ -0,0 +1,123 @@
+import { useValues } from 'kea'
+import { Sparkline } from 'lib/components/Sparkline'
+import { dayjs } from 'lib/dayjs'
+import { useCallback, useMemo } from 'react'
+
+import { themeLogic } from '~/layout/navigation-3000/themeLogic'
+import { ErrorTrackingIssueAggregations, ErrorTrackingSparklineConfig } from '~/queries/schema/schema-general'
+
+import { errorTrackingLogic } from '../errorTrackingLogic'
+import { sparklineLabels } from '../utils'
+
+export function OccurenceSparkline({
+    values,
+    unit,
+    interval,
+    className,
+    displayXAxis = false,
+}: {
+    values: number[]
+    unit: TimeUnit
+    interval: number
+    className?: string
+    displayXAxis?: boolean
+}): JSX.Element {
+    const colors = useSparklineColors()
+
+    const [data, labels, labelRenderer] = useMemo(() => {
+        return [
+            wrapDataWithColor(values, colors),
+            sparklineLabels({ value: interval, interval: unit } as ErrorTrackingSparklineConfig),
+            (label: string) => {
+                switch (unit) {
+                    case 'hour':
+                    case 'minute':
+                        return dayjs(label).format('D MMM YYYY HH:mm (UTC)')
+                    case 'day':
+                        return dayjs(label).format('D MMM YYYY (UTC)')
+                    case 'week':
+                        return dayjs(label).format('D MMM YYYY (UTC)')
+                    case 'month':
+                        return dayjs(label).format('MMM YYYY (UTC)')
+                    default:
+                        return dayjs(label).format('D MMM YYYY (UTC)')
+                }
+            },
+        ]
+    }, [values, unit, interval, colors])
+
+    const withXScale = useCallback((scale: ScaleOptions) => {
+        return {
+            ...scale,
+            type: 'timeseries',
+            ticks: {
+                display: true,
+                maxRotation: 0,
+                maxTicksLimit: 5,
+                font: {
+                    size: 10,
+                    lineHeight: 1,
+                },
+            },
+            time: {
+                unit: 'day',
+                displayFormats: {
+                    day: 'D MMM',
+                },
+            },
+        } as ScaleOptions
+    }, [])
+
+    return (
+        <Sparkline
+            className={className}
+            data={data}
+            labels={labels}
+            renderLabel={labelRenderer}
+            withXScale={displayXAxis ? withXScale : undefined}
+        />
+    )
+}
+
+function useSparklineColors(): { color: string; hoverColor: string } {
+    const { isDarkModeOn } = useValues(themeLogic)
+    return useMemo(() => {
+        return {
+            color: isDarkModeOn ? 'primitive-neutral-200' : 'primitive-neutral-800',
+            hoverColor: 'primary-3000',
+        }
+    }, [isDarkModeOn])
+}
+
+export function useSparklineData(aggregations?: ErrorTrackingIssueAggregations): [number[], TimeUnit, number] {
```

Comment:
Should this be taking an optional type? I think it's making the implementation of the component more complicated and causing us to do checks to see if there are any returned `values` in other components

```suggestion
export function useSparklineData(aggregations: ErrorTrackingIssueAggregations): [number[], TimeUnit, number] {
```